src/Tools/ExternalAccess/AspNetCore/ExternalAccessAspNetCoreResources.Designer.cs

```diff
@@ -0,0 +1,64 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.AspNetCore {
```

Comment:
😕 This file shouldn't be necessary (Arcade generates the designer files at build time)

Reply:
will remove in followup!